.ci/integrationTestDownstream.groovy

```diff
@@ -22,7 +22,7 @@ pipeline {
     DISABLE_BUILD_PARALLEL = "${params.DISABLE_BUILD_PARALLEL}"
   }
   options {
-    timeout(time: 1, unit: 'HOURS')
+    timeout(time: 2, unit: 'HOURS')
```

Comment:
we test 13 versions of the stack in some cases 1 hour is too tight.

TODO: check if we can rid of some of those stack tests (e.g. test only 7.5+)

Reply:
we can start removing some 

![image](https://user-images.githubusercontent.com/5400788/105210457-5adc6480-5b4b-11eb-912e-3997f1e72917.png)